Formula/m/morpheus.rb

```diff
@@ -0,0 +1,98 @@
+class Morpheus < Formula
+  desc "Modelling environment for multi-cellular systems biology"
+  homepage "https://morpheus.gitlab.io/"
+  url "https://gitlab.com/morpheus.lab/morpheus/-/archive/v2.3.7/morpheus-v2.3.7.tar.gz"
+  sha256 "ad5694a098e4752db53659ee983c3ae417a43747320e73c3005f6cf88b52d55c"
+  license "BSD-3-Clause"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+(?:_?\d+)?)$/i)
+  end
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "ninja" => :build
+  depends_on "ffmpeg"
+  depends_on "graphviz"
+  depends_on "libomp"
+  depends_on "libtiff"
+  depends_on "qt@5"
+
+  uses_from_macos "bzip2"
+  uses_from_macos "libxml2"
+  uses_from_macos "zlib"
+
+  def install
+    # has to build with Ninja until: https://gitlab.kitware.com/cmake/cmake/-/issues/25142
+    args = []
+    args << "-G Ninja"
```

Comment:
```suggestion
    args = ["-G Ninja"]
```

Reply:
Same as the style change [above](https://github.com/Homebrew/homebrew-core/pull/162155#discussion_r1562265144). This was added by @SMillerDev in https://github.com/Homebrew/homebrew-core/pull/162155#discussion_r1560576053 and already signed off for merging.

Reply:
Because we're all humans and carlocab brings up a good point, this is a nicer way of writing this.

Reply:
You need to apply https://github.com/Homebrew/homebrew-core/pull/162155#discussion_r1562266473 anyway, so may as well do this now